packages/beacon-state-transition/src/epoch/processEffectiveBalanceUpdates.ts

```diff
@@ -49,7 +55,17 @@ export function processEffectiveBalanceUpdates(state: CachedBeaconStateAllForks,
       // Should happen rarely, so it's fine to update the tree
       validators.get(i).effectiveBalance = effectiveBalance;
       // Also update the fast cached version
-      effectiveBalanceIncrement = Math.floor(effectiveBalance / EFFECTIVE_BALANCE_INCREMENT);
+      const newEffectiveBalanceIncrement = Math.floor(effectiveBalance / EFFECTIVE_BALANCE_INCREMENT);
+      const deltaEffectiveBalanceIncrement = newEffectiveBalanceIncrement - effectiveBalanceIncrement;
+      if (forkSeq > ForkSeq.phase0) {
+        const currentParticipation = (state as BeaconStateAltair).currentEpochParticipation.getAll();
```

Comment:
Is this slow? should we be using `state.currentEpochParticipation.get(i)` ?

Reply:
yes, will update it 👍